package.json

```diff
@@ -39,6 +39,7 @@
         "grunt-shell": "^2.1.0",
         "grunt-webpack": "^3.0.2",
         "grunt-zip": "^0.17.1",
+        "jsoneditor": "^5.9.3",
```

Comment:
This package actually belongs in `clients/web/src/package.json`, since it's a dependency of the web client sources. You can remove it from this file entirely, and add it to that one using the `~` version specifier (instead of `^`).